### ISSUE TITLE
fabric8 goal order creates faulty kubernetes.json

### ISSUE THREAD

**Issue by KurtStam** [fabric8io/fabric8#4721]
When running 

mvn clean install fabric8:json the output is different then running
mvn clean fabric8:json install

fabric8-maven-plugin:2.2.26:attach goal happens first; then the fabric8-maven-plugin:2.2.26:json is last.
I can reproduce the issue in the fabric8-ipaas/elastic project where in the log stash template is completely missing using the first maven command.

https://gist.github.com/KurtStam/8a56882c433ac212c847
https://gist.github.com/jstrachan/e420013faa4c169e0e93#file-elasticsearch-kubernetes-json-L132-L139

**Comment by davsclaus** [comment]
Yeah I think the order may be important. I usually run mvn clean install fabric8:json docker:build. 

But maybe we could have an uber maven goal that combines the other goals and execute them in correct order?

**Comment by davsclaus** [comment]
We provide a maven profile that runs the order correct. We recommend you add those and use that too
https://github.com/fabric8io/ipaas-quickstarts/blob/master/quickstart/java/simple-mainclass/pom.xml#L174
